projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/model/CaseDetail.kt

```diff
@@ -62,6 +62,8 @@ data class Offence(
     val id: String,
     val code: String,
     val description: String,
+    val mainCategoryDescription: String,
+    val subCategoryDescription: String,
```

Comment:
when we get these back from community api both fields are non-nullable so have followed suit here as my understanding is that it is the same underlying database. Should I do anything else to 100% confirm this or all good?